_configuration/configuration.md

```diff
@@ -606,7 +606,10 @@ Setting these too high will cause the corresponding stepper motor to lose steps,
 ```
 When the velocity of any axis changes, its acceleration (or deceleration) in mm/s/s is limited by the current max acceleration setting. Also see the *jerk* settings below, which specify the largest instant speed change that can occur between segments.
 
-A value of 3000 means that an axis may accelerate from 0 to 3000mm/m (50mm/s) within a one second movement.
+A value of e.g 50 means that an axis can accelerate from 0 to 50 mm/s in one second. 
```

Comment:
No need for e.g here. But, if we decide to keep it, it should be parenthetical:
```
A value of (e.g.) 50...
```
Usually we'd just use
```
A value of 50, for example, ...
```